docs/automated-workflows/dagman-using-other-dags.rst

```diff
@@ -1,338 +1,319 @@
 Composing workflows from multiple DAG files
 ===========================================
```

Comment:
Where possible, I'd like to keep these titles fitting on a single line on the left hand side navigation menu.  WIth that, how about changing this to 

Composing multiple DAGs
======================